CHANGELOG.md

```diff
@@ -3,23 +3,19 @@ All notable changes to this project are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). See the [CONTRIBUTING guide](./CONTRIBUTING.md#Changelog) for instructions on how to add changelog entries.
 
-## [Unreleased 5.0.x]
+## [Unreleased 6.0.0]
 ### Added
-- New quality check workflows [(#753)](https://github.com/wazuh/wazuh-indexer/pull/753)
+
 ### Dependencies
 
 ### Changed
-- Update opensearch.yml template and auto-generate creds directory [(#755)](https://github.com/wazuh/wazuh-indexer/pull/755)
-- Update certificates directory ownership and permissions [(#766)](https://github.com/wazuh/wazuh-indexer/pull/766)
-- Improve DEB and RPM packaging scripts [(#805)](https://github.com/wazuh/wazuh-indexer/pull/805)
 
 ### Deprecated
 
 ### Removed
 
 ### Fixed
-- Fix NPE in node stats due to QueryGroupTasks ([#17576](https://github.com/opensearch-project/OpenSearch/pull/17576))
 
 ### Security
 
-[Unreleased 5.0.x]: https://github.com/wazuh/wazuh-indexer/compare/4.12.0...main
+[Unreleased 6.0.0]: https://github.com/wazuh/wazuh-indexer/compare/6.0.0...main
```

Comment:
I think the first version here should be the last _release_. Check [4.13.0](https://github.com/wazuh/wazuh-indexer/pull/823/files).
```suggestion
[Unreleased 6.0.0]: https://github.com/wazuh/wazuh-indexer/compare/4.12.0...main
```

Reply:
`main` is now 5.0.0, so we need to compare from 5.0.0 to 6.0.0

> https://github.com/wazuh/wazuh-indexer/compare/main...6.0.0